src/pages/Widget/Preview.tsx

```diff
@@ -11,9 +13,12 @@ export default function Preview({ classes = "" }) {
 
   return (
     <section className={`${classes} @container/preview pb-4`}>
-      <h2 className="text-lg @4xl/widget:text-2xl text-center @4xl/widget:text-left mb-3">
-        That's what our Donation Form looks like:
-      </h2>
+      <span className="max-sm:hidden">
+        <p className="flex text-navy-d4 text-2xl font-gochi text-nowrap">
+          <Image src={character} className="h-[45px] mr-2 pb-2" />
+          Check out the LIVE preview of your Donation Form!
```

Comment:
this text doesn't have enough space around 
<img width="656" alt="image" src="https://github.com/AngelProtocolFinance/angelprotocol-web-app/assets/89639563/10d2a490-3a04-4b55-a841-ac627feeb847">

original 
<img width="365" alt="image" src="https://github.com/AngelProtocolFinance/angelprotocol-web-app/assets/89639563/c594b179-0399-4727-94bb-f5f4a35b2ec9">

Reply:
@ap-justin Good catch! I'll fix that.

Reply:
Fixed in 4609260c. Also fixed issues with the new text not wrapping & the snippet alignment with it's title.